cub/warp/warp_exchange.cuh

```diff
@@ -468,5 +468,89 @@ public:
   //@}  end member group
 };
 
+template <typename InputT, int ITEMS_PER_THREAD>
+class WarpExchangeRegister
+{
+  static_assert(PowerOfTwo<ITEMS_PER_THREAD>::VALUE, "ITEMS_PER_THREAD must be a power of two");
+
+  template <int SIZE, int idx>
```

Comment:
We prefer to capitalize template parameters: 
```suggestion
  template <int SIZE, int IDX>
```

Reply:
sure, I just used lowercase to indicates a subscript index, but I'll change that